Examples/Scripts/Python/geometry.py

```diff
@@ -72,6 +72,7 @@ def runGeometry(
                 processRepresenting=True,
                 processBoundaries=True,
                 processVolumes=True,
+                processNonMaterial=True,
```

Comment:
Yes that flag has always existed. If you want to use geometry.py for the material mapping this flag need to be here and it was use in the equivalent example (WriteTrackingGeometry). I thought it would be good to add this here in preparation of moving the mapping doc to the Python bindings